easybuild/tools/options.py

```diff
@@ -252,6 +252,8 @@ def basic_options(self):
         opts = OrderedDict({
             'dry-run': ("Print build overview incl. dependencies (full paths)", None, 'store_true', False),
             'dry-run-short': ("Print build overview incl. dependencies (short paths)", None, 'store_true', False, 'D'),
+            'continue': ("Continue the installation from certain step",
+                     'choice', 'store_or_None', SOURCE_STEP, 'c', all_stops),
```

Comment:
Not sure if we want `eb -c` to be equivalent with `eb --continue`, maybe we should save up `-c` for something else?

It seems to me that `--continue` will always be a bit 'risky' (it may fail for all kinds of reasons that are going to be hard to fix)...